content/en/user-guide/web-application/extensions-library/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Extensions Library"
+weight: 5
+description: Get started with Extensions Library to extend LocalStack by adding new services and features via Extension mechanism.
```

Comment:
maybe just a `Extend LocalStack by adding new services and features as extensions`?